### DISCUSSION TITLE
Swapping windows in a predefined layout ?

### DISCUSSION THREAD

**Discussion by bassocksky** [rxhanson/Rectangle#1517] [General]
I work on MacOs, Linux, BSDs with keyboard driven tiling, mostly using terminal windows.
On Linux and BSDs I have the possibility - with the right chosen windows managers - to define the following window management workflow:

- open a bunch of terminals
- force them to a given layout (for example one big Window on the left two thirds and the remaining windows have to share the rest of the space at the right side ).
- focus one of the small windows and swap this window with the big one of the left side by a single key stroke combination.

If I use four windows with an "two-third and three ninth window" layout I can simple set the initial arrangement,
but I didn't find any method to swap windows in the settings.

Maybe I'm too stupid to find it ...

Is this generally possible?

best regards
Andre
aka bassocksky

**Comment by rxhanson** [comment]
I’ve had plans to add more auto-tiling behavior like this to Rectangle Pro, but it hasn’t bubbled up to top priority yet. I’ll add your +1. Thanks!
